cypress/support/step-definitions/show-edit-pod-steps.js

```diff
@@ -38,20 +35,18 @@ When('Edit icon has color {string}', (color) => {
   icon().should('have.css', 'color', color);
 });
 
-Then('Show Edit Pod component has border property', () => {
-  showEditPodPreview()
-    .should('not.have.class', `${SHOW_EDIT_POD_BLOCK_CONTENT}no-border`)
-    .and('have.css', 'border', '1px solid rgb(204, 214, 219)');
-  showEditPodPreview().children()
-    .should('not.have.class', SHOW_EDIT_POD_NO_BORDER_CONTENT);
+Then('Show Edit Pod component has border {string} color', (color) => {
+  showEditPodComponent().find('div').eq(0).should('have.css', 'border-bottom-color', color);
+  showEditPodComponent().find('div').eq(0).should('have.css', 'border-left-color', color);
+  showEditPodComponent().find('div').eq(0).should('have.css', 'border-right-color', color);
+  showEditPodComponent().find('div').eq(0).should('have.css', 'border-top-color', color);
 });
 
-Then('Show Edit Pod component has no border property', () => {
-  showEditPodPreview()
-    .should('have.class', `${SHOW_EDIT_POD_BLOCK_CONTENT}no-border`)
-    .and('have.css', 'border', '0px none rgba(0, 0, 0, 0.85)');
-  showEditPodPreview().children()
-    .should('have.class', SHOW_EDIT_POD_NO_BORDER_CONTENT);
+Then('Show Edit Pod component has no border {string} color', (color) => {
+  showEditPodComponent().find('div').eq(0).should('have.css', 'border-bottom-color', color);
```

Comment:
what is the difference between this TC and the `Show Edit Pod component has border {string} color` ?

Reply:
fixed